kuflow-rest/etc/openapi/readme.md

```diff
@@ -27,7 +27,7 @@ python: true
 title: KuFlow
 override-client-name: KuFlowRestClient
 
-input-file: https://raw.githubusercontent.com/kuflow/kuflow-openapi/5bc6e6e6063f87504920fbe8cba3e54f71797504/specs/api.kuflow.com/v2022-10-08/openapi.yaml
+input-file: https://raw.githubusercontent.com/kuflow/kuflow-openapi/336b24183db579748802c9170286dece8d1a64fa/specs/api.kuflow.com/v2022-10-08/openapi.yaml
```

Comment:
Warning is not the same as JAVA SDK.
```suggestion
input-file: https://raw.githubusercontent.com/kuflow/kuflow-openapi/52afccce9633ed3eb6f4ae78c50e4642ec424857/specs/api.kuflow.com/v2022-10-08/openapi.yaml
```

Reply:
yes, it was a mistake coping the URL, this is the latest one.